shared/src/business/entities/NewPetitionerUser.ts

```diff
@@ -25,8 +25,9 @@ export class NewPetitionerUser extends JoiValidationEntity {
       .messages({ '*': 'Passwords must match' }),
     email: JoiValidationConstants.EMAIL.required()
       .messages({
-        '*': 'Enter a valid email address',
-        'string.max': 'Email address must contain fewer than 100 characters', //todo test
```

Comment:
Not really anything to do with this PR, but is there a reason we don't enforce character limits for other email cases?

Reply:
E.g., I wonder if it eventually makes sense to have a generic email validation/messages thing for emails across the system.